driver/config/config_test.go

```diff
@@ -61,20 +61,18 @@ func TestViperProvider(t *testing.T) {
 					config.ViperKeySelfServiceLoginUI:        "http://test.kratos.ory.sh/#/login",
 					config.ViperKeySelfServiceSettingsURL:    "http://test.kratos.ory.sh/#/settings",
 					config.ViperKeySelfServiceRegistrationUI: "http://test.kratos.ory.sh/#/register",
-					config.ViperKeySelfServiceErrorUI:        "http://test.kratos.ory.sh/#/error",
+					config.ViperKeySelfServiceErrorUI:        "/error",
```

Comment:
Please add new tests and cases instead of modifying existing ones that should still pass.